garageScript/c0d3-app
# fix: Unlink/Connect discord buttons appears for any profiles

Closes #2279 

Currently, when you access a profile that isn't yours, it looks like this:

<details>
<summary>
Image
</summary>
<img width="461" alt="image" src="https://user-images.githubusercontent.com/35906419/190668927-26fbd031-5cd5-47e6-98a5-9275966d9f57.png">
</details>

The buttons show up even though it's not your profile. Merging this PR will fix it by comparing the username in the session to the profile's user username.

<details>
<summary>
Image
</summary>
<img width="452" alt="image" src="https://user-images.githubusercontent.com/35906419/190669217-5e289a2f-b166-4f06-8316-8262f06bcfae.png">
</details>

To test it:

1. Go to `/profile/leetcoder`
2. Notice how the `CONNECT TO DISCORD` button doesn't appear
